libs/components/text-editor/src/lib/modules/text-editor/url-modal/text-editor-url-modal.component.html

```diff
@@ -26,7 +26,11 @@
             />
           </sky-input-box>
         </div>
-        <sky-input-box stacked="true" class="sky-form-group">
+        <sky-input-box
+          *ngIf="!openLinksInNewWindowOnly; else onlyOpenLinksInNewWindow"
```

Comment:
Since we're allowing consumers to mix and match `new` and `existing` as options, we should also add logic for when consumers only specify `existing`. If a consumer only specifies `existing`, I think it's safe to hide the element that lets them select the option and not show any help text since that's the default behavior of a link with no `target` attribute.

Reply:
Added logic in. Thanks for the suggestion.